### PR TITLE
schemas: Document all fields and query options

### LINKED ISSUE #13: Extend schemas with human-readable descriptions to improve docs

### PR DESCRIPTION
**Pull request by tchx84**
Closes #13

### CODE REVIEW

File: app/schemas.py [hunk 1]
```diff
@@ -22,14 +22,46 @@
...
+    )
+    pm1dot0: Optional[float] = Field(
+        None,
+        title="PM1.0",
+        description="Inhalable particles that are 1.0 micrometers and smaller",
+    )
+    pm2dot5: Optional[float] = Field(
+        None,
+        title="PM2.5",
+        description="Inhalable particles that are 2.5 micrometers and smaller",
```
Comment (garyservin) [inline app/schemas.py:43]: The actual measure is micograms per cube meter of particles of the specific size

Comment (tchx84) [reply app/schemas.py:43]: Where did you take that definition? I took this from https://www.epa.gov/pm-pollution/particulate-matter-pm-basics#PM

Comment (garyservin) [reply app/schemas.py:43]: PM2.5 defines the particles size, but the actual measurement we record from the sensor is the concentration in ug/m3. This is from the [datasheet](https://www.espruino.com/datasheets/PMS7003.pdf) page 13, Data2 field

Comment (tchx84) [reply app/schemas.py]: How about  now? maybe is redundant but easier to understand(?)

**Review by garyservin — APPROVED** [review]
LGTM

**Pull request merged**